README.md

```diff
@@ -53,66 +53,35 @@ PDFXKit as a dependency to your `Podfile`.
 ``` Ruby
 # Replace `YourAppName` with your app's target name.
 target :YourAppName do
-  use_frameworks!
-
-  pod 'PSPDFKit', podspec: 'https://customers.pspdfkit.com/pspdfkit-ios/latest-framework.podspec'
```

Comment:
Good point! The integration is now implicit because PSPDFKit is a dependency in the podspec (https://github.com/PSPDFKit/PDFXKit/pull/9/files#diff-6f2cb93d716e8580740436aa50d12b4ca997d27f1ebe7c02ea1ba8e5d4d1a47b).

I'll add back the pod in the README.

Reply:
We could also just change the description (i.e., remove "You'll have to add PSPDFKit as well as PDFXKit as a dependency to your Podfile") and keep the simpler form.